example/javascriptlib/testing/3-integration-suite-cypress/client/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
```

Comment:
Is this a necessary part of the example?

Reply:
nah, taking it out